### PR TITLE
samples: profiler: Add sample

### PR DESCRIPTION
**Pull request by MarekPieta**
Change adds Profiler sample.

### CODE REVIEW

**Comment by MarekPieta** [comment]
@pdunaj, @Qbicz , could you review?
@ru-fu , could you review documentation? (references to Profiler page will be added after merging Profiler docs)

**Comment by MarekPieta** [comment]
thanks, @ru-fu, changes were all ok - I squashed it
Links wil be added after profiler documentation

File: samples/profiler/README.rst [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+  There are several numerical values associated with this event.
+  The values are updated periodically.
+
+
+Requirements
+************
+
+* One of the following development boards:
+
+  * nRF9160 DK board (PCA10090)
```
Comment (pdunaj) [inline samples/profiler/README.rst]: does not match platform whitelist `nrf52_pca10040 nrf52840_pca10056`

Comment (MarekPieta) [reply samples/profiler/README.rst]: Ok, I will add it to whitelist

**Comment by MarekPieta** [comment]
@pdunaj, can we merge ?

**Pull request merged**